dd-sdk-android/src/test/kotlin/com/datadog/android/core/internal/SdkFeatureTest.kt

```diff
@@ -291,16 +301,19 @@ internal class SdkFeatureTest {
         )
     }
 
-    @Test
-    fun `𝕄 do nothing 𝕎 withWriteContext(callback) { no Datadog context }`() {
+    @ParameterizedTest
```

Comment:
Same as above, `BoolForgery` is enough, no need for the parametrized test here and in other test files where `ValueSource` is just true or false.